.github/workflows/prebuild.yml

```diff
@@ -31,6 +31,17 @@ jobs:
       - run: ./scripts/check-dco.sh
       - run: PATH=$PATH:$(pwd) ./scripts/check-flatc.sh
   
+  git-secrets:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Install git secrets
+        run: sudo apt-get update && sudo apt install -y git-secrets
```

Comment:
Don't need sudo

Reply:
Without sudo workflow fails with permission denied. See https://github.com/austinvazquez/soci-snapshotter/actions/runs/8485926066/job/23251567287?pr=17

Reply:
We can set PREFIX to a non-privileged location if we really want to avoid `sudo`. It's already inside a container, so I don't really care, just making a note of it